types/nightwatch/index.d.ts

```diff
@@ -6500,34 +6661,37 @@ export interface WebDriverProtocolUserActions {
      *   }
      * }
      *
+     * @see https://nightwatchjs.org/api/clickAndHold.html#apimethod-container
      */
     clickAndHold(
         selector: string,
-        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<void>) => void,
-    ): this;
+        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<null>) => void,
+    ): Awaitable<this, null>;
     clickAndHold(
         using: LocateStrategy,
         selector: string,
-        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<void>) => void,
-    ): this;
+        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<null>) => void,
+    ): Awaitable<this, null>;
 
     /**
      * Release the depressed left mouse button at the current mouse coordinates (set by `.moveTo()`).
      *
      */
-    releaseMouseButton(callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<void>) => void): this;
+    releaseMouseButton(callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<null>) => void): Awaitable<this, null>;
 
     /**
      * Click at the current mouse coordinates (set by `.moveTo()`).
      *
      * The button can be (0, 1, 2) or ('left', 'middle', 'right'). It defaults to left mouse button.
      *
+     * @see https://nightwatchjs.org/api/mouseButtonClick.html
+     *
      * @deprecated
```

Comment:
Add the message -- Please use the new User Actions API instead.

^ same for other deprecated commands below as well.